packages/markdown/src/lib/remark-slate/remarkDefaultElementRules.ts

```diff
@@ -80,10 +80,14 @@ export const remarkDefaultElementRules: RemarkElementRules = {
           indent = 1
         ) => {
           _node.children?.forEach((listItem) => {
+            const defaultType = options.editor.getType({ key: 'p' });
+            const todoListType = options.editor.getType({ key: 'action_item' }) ?? defaultType
```

Comment:
Thanks for the PR! We don't use `TodoListPlugin` with indent list, hence it can't be in this `if (options.indentList)` condition. Would love a unit test!

Reply:
Firstly, I would like to confirm whether TodoListPlugin will be applied in the indentList? Why are there no checks for mutually exclusive plugins, such as indentListPlugin and listPlugin

Reply:
I'm sorry to say that it took me some time to understand the relationship between indentListPlugin and listPlugin, but my user data has been contaminated and I have to go extra and clean up the old data

Reply:
I may have made the same mistake again. I didn't understand the connection between todoPlugin and indentList, and I have been confused about why there is a checked judgment in indentList. So, please clearly tell me that indentList=list+todoPlugin

Reply:
Either use https://platejs.org/docs/list **or** https://platejs.org/docs/indent-list